db/migrate/20230120235417_creates_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreatesComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :post, foreign_key: true, index: true
+      t.references :author, foreign_key: true, index: true
```

Comment:
- [ ] I'm happy to see that the name of a user will be called author as shown on the ERD diagram, but we should let Ruby know which table is an author we are referring to by explicitly adding `foreign_key: { to_table: 'users' }`